iota/transaction/base.py

```diff
@@ -104,9 +112,9 @@ def __init__(
     Can be negative (i.e., for spending inputs).
     """
 
-    self.tag = tag # type: Optional[Tag]
+    self.legacy_tag = legacy_tag # type: Optional[Tag]
     """
-    Optional classification tag applied to this transaction.
+    Optional classification legacy_tag applied to this transaction.
     """
 
     self.nonce = nonce # type: Optional[Hash]
```

Comment:
This one may need some love as well — `Hash` automatically pads itself to 81 trytes.